packages/snap-preact-components/src/components/Atoms/NoResults/NoResults.tsx

```diff
@@ -0,0 +1,132 @@
+/** @jsx jsx */
+import { Fragment, h } from 'preact';
+
+import { observer } from 'mobx-react-lite';
+import { jsx, css } from '@emotion/react';
+import classnames from 'classnames';
+
+import { Theme, useTheme } from '../../../providers';
+import { cloneWithProps, mergeProps } from '../../../utilities';
+import { ComponentProps, StylingCSS } from '../../../types';
+import { handleize } from '@searchspring/snap-toolbox/dist/cjs/filters/handleize';
+import { SearchController } from '@searchspring/snap-controller';
+
+const CSS = {
+	noresults: () => css({}),
+};
+
+export const NoResults = observer((properties: NoResultsProps): JSX.Element => {
+	const globalTheme: Theme = useTheme();
+
+	const defaultProps: Partial<NoResultsProps> = {
+		dymText: `Did you mean <a href=${properties.controller?.store?.search?.didYouMean?.url.href}>${properties.controller?.store?.search?.didYouMean?.string}</a>?`,
+		suggestionsTitleText: `Suggestions`,
+		suggestionsList: [
+			`Check for misspellings.`,
+			`Remove possible redundant keywords (ie. "products").`,
+			`Use other words to describe what you are searching for.`,
+		],
+		contactsTitleText: `Still can't find what you're looking for? <a href="/contact-us">Contact us</a>.`,
+		contactsList: [
+			{
+				title: `Address`,
+				content: `123 Street Address<br />City, State, Zipcode`,
+			},
+			{
+				title: `Hours`,
+				content: `Monday - Saturday, 00:00am - 00:00pm<br />Sunday, 00:00am - 00:00pm`,
+			},
+			{
+				title: `Phone`,
+				content: `<a href="tel:1234567890">123-456-7890</a>`,
+			},
+			{
+				title: `Email`,
+				content: `<a href="mailto:email@site.com">email@site.com</a>`,
+			},
+		],
+	};
+
+	const props = mergeProps('noResults', globalTheme, defaultProps, properties);
+
+	const {
+		staticSlot,
+		suggestionsTitleText,
+		dymText,
+		suggestionsList,
+		hideContact,
+		contactsTitleText,
+		contactsList,
+		controller,
+		disableStyles,
+		className,
+		style,
+	} = props;
+
+	const dym = controller?.store?.search?.didYouMean;
+
+	const styling: { css?: StylingCSS } = {};
+	if (!disableStyles) {
+		styling.css = [CSS.noresults(), style];
+	} else if (style) {
+		styling.css = [style];
+	}
+
+	return (
+		<div className={classnames('ss__no-results', className)}>
+			{staticSlot ? (
```

Comment:
I'm not sure the static slot should exclude usage of the other props...

Reply:
The slot should probably have a wrapper div to handle changing the order via CSS as needed.